common/api/core-frontend.api.md

```diff
@@ -5839,7 +5838,7 @@ export class ImageryMapLayerFormat extends MapLayerFormat {
     static createMapLayerTree(layerSettings: ImageMapLayerSettings, layerIndex: number, iModel: IModelConnection): MapLayerTileTreeReference | undefined;
 }
 
-// @internal
+// @beta

```

Comment:
You have promoted `resetTreeOwner` to beta, but it is `internal` in the base `TileTreeReference` class.

Reply:
This was an oversight and I have reverted since it seems like it should stay internal, and the correct publicly accessible way to do this is through the [viewport](https://github.com/iTwin/itwinjs-core/blob/47d893ee815788ab1d6cbad681604b86d19f85da/core/frontend/src/Viewport.ts#L896)